packages/terra-date-time-picker/tests/wdio/date-time-picker-spec.js

```diff
@@ -691,4 +691,40 @@ Terra.describeViewports('DateTimePicker', ['tiny', 'large'], () => {
 
     Terra.it.validatesElement();
   });
+
+  describe('With timezone', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-time-picker/date-time-picker/date-time-picker-with-timezone'));
+
+    Terra.it.matchesScreenshot();
```

Comment:
`Terra.it.*` have been deprecated and replaced by `Terra.validates.*` See https://github.com/cerner/terra-toolkit-boneyard/blob/main/docs/Wdio_Utility.md#test-assertion-helpers

Reply:
I think either @saket2403  or @pranav300 will have a separate PR for this.